python/osrd_schemas/osrd_schemas/rolling_stock.py

```diff
@@ -234,40 +234,16 @@ class TowedRollingStock(BaseModel, extra="forbid"):
     all the traction properties.
     """
 
-    railjson_version: RAILJSON_ROLLING_STOCK_VERSION_TYPE = Field(default=RAILJSON_ROLLING_STOCK_VERSION)
     name: str = Field(max_length=255)
+    railjson_version: RAILJSON_ROLLING_STOCK_VERSION_TYPE = Field(default=RAILJSON_ROLLING_STOCK_VERSION)
     locked: bool = Field(default=False, description="Whether the rolling stock can be edited/deleted or not")
-    effort_curves: EffortCurves = Field(description="Curves mapping speed (in m/s) to maximum traction (in newtons)")
-    base_power_class: Optional[str] = Field(
-        description="The power usage class of the train (optional because it is specific to SNCF)", default=None
-    )
-    power_restrictions: Optional[PowerRestrictions] = Field(
-        description="Mapping from train's power restriction codes to power classes", default=None
-    )
-    length: PositiveFloat = Field(description="The length of the train, in m")
-    max_speed: PositiveFloat = Field(description="Maximum speed in m/s")
-    startup_time: float = Field(ge=0, description="The time the train takes before it can start accelerating in s")
-    startup_acceleration: float = Field(ge=0, description="The maximum acceleration during startup in m/s^2")
+    mass: NonNegativeFloat = Field(description="The mass of the train, in kg")
+    length: NonNegativeFloat = Field(description="The length of the train, in m")
     comfort_acceleration: PositiveFloat = Field(description="The maximum operational acceleration in m/s^2")
```

Comment:
```suggestion
    comfort_acceleration: NonNegativeFloat = Field(description="The maximum operational acceleration in m/s^2")
```

Reply:
Adressed in [e3665de](https://github.com/OpenRailAssociation/osrd/pull/9436/commits/e3665de80e3cca07ddb30af3ad578d85c2055d08) (and also some of your general comments).